### ISSUE TITLE
Select filter problem

### ISSUE THREAD

**Issue by mssimi** [stwe/DatatablesBundle#483]
Hey, I need select filter where options are:

any
IS NULL
IS NOT NULL

I am unable to find way to do that with current search_types.

**Comment by mleczakm** [comment]
👍

**Comment by stwe** [comment]
This is solved in version 1.0.

```
                'filter' => array(SelectFilter::class, array(
                    'select_search_types' => array(
                        '' => null, // never used placeholder
                        '2' => 'like',
                        'send_isNull' => 'isNull',
                        'send_isNotNull' => 'isNotNull'
                    ),
                    'select_options' => array(
                        '' => 'Any', 
                        '2' => 'Title with 2',
                        'send_isNull' => 'is Null',
                        'send_isNotNull' => 'is not Null'
                    ),
                ))
```
